### DISCUSSION TITLE
Error log in submission

### DISCUSSION THREAD

**Discussion by begovegar** [PaddlePaddle/PaddleSpatial#245] [BaiduKDDCup2022] [answered]
Hi, can we see the complete log of this error?
We are Innova team and we would need the log of the submission at 2022-07-08 05:05
![image](https://user-images.githubusercontent.com/104911705/177927600-aa709c8b-9483-4fc5-9a70-86e436aa1044.png)

**Comment by shaido987** [comment, marked as answer]
This error seems to be raised when the accuracy of a prediction is too low, you can refer to the code here for the full error message:  

https://github.com/PaddlePaddle/PaddleSpatial/blob/ae70ae2a00a14dcef67a57c34a79e3791b8d0515/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle/evaluation.py#L244-L251

(note that I'm just a participant and not one of the organizers so I could be wrong as I don't have access to any logs, etc.)

**Comment by begovegar** [comment]
Thanks for your quick response shaido987!

**Reply by shaido987** [reply]
No problems :)
